app/Http/Requests/ProjectRequest.php

```diff
@@ -37,7 +37,7 @@ public function rules()
             'initiative_category_otther' => 'nullable',
             'budget' => 'required|integer|gte:0',
             'currency' => 'required|max:3',
-            'exchange_rate' => 'required|numeric',
+            'exchange_rate' => 'sometimes',
```

Comment:
This needs a 'required' as well, otherwise it's considered optional when it is shown/enabled.